subprojects/pcre2.wrap

```diff
@@ -0,0 +1,9 @@
+[wrap-file]
+directory = pcre2-10.23
+source_url = https://sourceforge.net/projects/pcre/files/pcre2/10.23/pcre2-10.23.zip
```

Comment:
Sourceforge has a tendency to be unreliable

the source is also available from on Githubhttps://github.com/PhilipHazel/pcre2/tree/pcre2-10.23  but it has a different structure than the patch requires